src/auth.ts

```diff
@@ -45,6 +46,7 @@ const roleScopes: Record<Role, Scope[]> = {
         Scope['read:healthcareprofessionals'], Scope['write:healthcareprofessionals'], Scope['delete:healthcareprofessionals'],
         Scope['read:facilities'], Scope['write:facilities'], Scope['delete:facilities'],
         Scope['read:submissions'], Scope['write:submissions'], Scope['delete:submissions'],
+        Scope['create:submissions'],
```

Comment:
no need to add create permission to any of the roles as `write` should already include this. `Create` is `write` without `update`.